packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -107,6 +107,27 @@ class _AnimatedState extends State<AnimatedWidget> {
 /// [textDirection] is provided, then the offsets are applied in the reading
 /// direction, so in right-to-left text, positive x offsets move towards the
 /// left, and in left-to-right text, positive x offsets move towards the right.
+///
+/// Here's an illustration of the [SlideTransition] widget, with it's [position]
+/// animated by a [CurvedAnimation] set to [Curves.elasticIn]:
+/// {@animation 300 378 https://flutter.github.io/assets-for-api-docs/assets/widgets/slide_transition.mp4}
+///
+/// See also:
+///
+///  * [AlignTransition], an animated version of an [Align] that animates its
+///    [Align.alignment] property.
+///  * [DecoratedBoxTransition], a widget that animates the different
+///    properties of a [DecoratedBox].
+///  * [FadeTransition], a widget that animates the opacity of a widget.
+///  * [PositionedTransition], a widget that animates its child from a start
+///    position to an end position over the lifetime of the animation.
+///  * [RelativePositionedTransition], a widget that transitions its child's
+///    position based on the value of a rectangle relative to a bounding box.
+///  * [RotationTransition], a widget that animates the rotation of a widget.
+///  * [ScaleTransition], a widget that animates the scale of a transformed
+///    widget.
+///  * [SizeTransition], a widget that animates its own size and clips and
+///    aligns its child.
```

Comment:
i think we should be a bit more conservative in our cross-links here. For example, I don't see how SlideTransition really relates to DecoratedBoxTransition or FadeTransition (the others seem more related).